example_app/lib/main.dart

```diff
@@ -0,0 +1,130 @@
+import 'package:flutter/material.dart';
+
+import 'package:avo_inspector/avo_inspector.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+AvoInspector? avoInspector;
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    AvoInspector.enableLogs(true);
+
+    avoInspector = AvoInspector(
+        apiKey: "my api key",
+        env: AvoInspectorEnv.dev,
+        appVersion: "",
+        appName: "");
+    avoInspector
+        ?.trackSchemaFromEvent(eventName: "App Open", eventProperties: {});
+
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        // This is the theme of your application.
+        //
+        // Try running your application with "flutter run". You'll see the
+        // application has a blue toolbar. Then, without quitting the app, try
+        // changing the primarySwatch below to Colors.green and then invoke
+        // "hot reload" (press "r" in the console where you ran "flutter run",
+        // or simply save your changes to "hot reload" in a Flutter IDE).
+        // Notice that the counter didn't reset back to zero; the application
+        // is not restarted.
```

Comment:
Should we remove this comment and the rest of the default ones to make the Avo related code stand out more?